accounts/platform/iam_platform_ci.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_iam_role_policy" "platform_ci" {
+  role   = module.aws_account.ci_role_name
+  policy = data.aws_iam_policy_document.platform_ci.json
+}
+
+data "aws_iam_policy_document" "platform_ci" {
+  dynamic "statement" {
+    for_each = [
+      "arn:aws:s3:::wellcomecollection-edge-lambdas"
```

Comment:
I assume these are being uploaded to a different bucket to our usual Lambdas because they need to live in us-east-1 with CloudFront?

Reply:
Yes.